code/controllers/CronTaskController.php

```diff
@@ -9,21 +10,11 @@ class CronTaskController extends Controller
 {
 
     /**
-     * If this controller is in quiet mode
+     * Setting this to true will make the controller print debug information
      *
      * @var bool
      */
-    protected $quiet = false;
```

Comment:
on that note, setQuiet() api was broken https://github.com/silverstripe-labs/silverstripe-crontask/pull/18/files#diff-875654ae714eb9197cc8215bb4372ec6L23